golem-cluster.yaml

```
@@ -51,10 +51,10 @@ provider:
           duration_hours: 0.5 # 30 minutes
 
           # Amount of GLMs for expected usage which Golem provider offer will be rejected if exceeded.
-          # Max cost is calculated as a sum of:
-          # - start_price / duration / cpu count
-          # - (env_per_hour_price * duration ) / cpu count
-          # - cpu_per_hour_price / duration / cpu_load
+          # Per cpu max cost is calculated as a sum of:
+          # - start_price / cpu_count
+          # - env_per_hour_price * duration / cpu_count
+          # - cpu_per_hour_price * duration * cpu_load 
```

isn't this _also_ per cpu?


```suggestion
          # - cpu_per_hour_price * duration * cpu_load / cpu_count
```

---

@shadeofblue cpu_per_hour_price is already per cpu - I don't think we should divide it again